### ISSUE TITLE
Hololens Button Press not working while using Hand Physics Service

### ISSUE THREAD

**Issue by A-Ivan** [microsoft/MixedRealityToolkit-Unity#10393]
## Describe the bug

My application requires both [pressable buttons](https://docs.microsoft.com/en-us/windows/mixed-reality/mrtk-unity/features/ux-building-blocks/button?view=mrtkunity-2021-05), for specific button use, and [ hand physics service (component type)](https://docs.microsoft.com/en-us/windows/mixed-reality/mrtk-unity/features/extensions/hand-physics-service?view=mrtkunity-2021-05), to detect collision between the user's hands and virtual objects.

My button is based on the prefab "[HandMenu_Small_HideOnHandDrop](https://github.com/microsoft/MixedRealityToolkit-Unity/tree/main/Assets/MRTK/Examples/Common/Prefabs)", located in Packages\MixedRealityToolkitExamples\Common\Prefabs. The button appears when the user shows their hand palm to the hololens camera.

When I do not use the hand physics service (MixedRealityToolkit > MixedRealityToolkitConfigurationProfile > Extensions >ServiceProvider > Microsoft.MixedReality.Toolkit.Extensions.HandPhysics > HandPhysicsService), I am able to press the buttons (I can catch their OnClick() calls).

The problem is that when I do use/enable the hand physics service the pressable buttons do not work (OnClick() is not called).

Is there any extra setup I need to do to use both hand physics service and pressable buttons at the same time? It isn't very clear from the documentation.

I've thought of enabling the hand physics service only after the button has been pressed, but in my case I would like the user to be able to press the button perform actions throughout their use, so the button press would not be something that occurs only in the beginning.

## To reproduce

1. Create a new scene
2. Add HandPhysicsService
3. Setup MRTK canvas
3. Add the "HandMenu_Small_HideOnHandDrop" to the canvas
4. Configure button OnClick() events
5. Run and try pressing the buttons

## Expected behavior

Button should be pressable even if the hand physics service is enabled. I expect the button to behave the same way as if the hand physics service was not enabled (be pressable), but not necessarily push it so that the button translates/flyes off, as the button is tied to the user's palm.

## Your setup (please complete the following information)

- Unity Version 2020.3.24f1
- MRTK Version 2.7.3

## Target platform (please complete the following information)

- HoloLens 2

**Comment by RogPodge** [comment]
Filing as a documentation task. I'm not sure we'll be able to dig deep into this in the near term due to v3 priorities.

**Comment by filip0910** [comment]
I have the same problem.

**Comment by JosiSiah** [comment]
I may have found a solution...

Essentially, I realized it is the **Right_PokePointer** generated in the hierarchy that interacts with the UX Buttons (Specifically i am using the **HandMenu_Small_HideOnHandDrop** prefab found in MRTK Examples).

![image](https://user-images.githubusercontent.com/118937840/203645342-b3ab4764-acc6-4b83-afc2-b4156dce8d68.png)

Inspecting the **PokePointer** script in this object I found that the **PokeLayerMasks** determine which layers the **PokePointer** can interact with. 

![image](https://user-images.githubusercontent.com/118937840/203646118-efee470a-5932-4805-b529-5e4b557cb4df.png)

Im not sure how to change which layers by default will be enabled/disabled in Poke Layer Masks. But i noticed that **Ignore Raycast** is unchecked by default.

![image](https://user-images.githubusercontent.com/118937840/203646719-099430a3-416c-402c-8e01-85cfceb44f28.png)

So, for a quick fix I placed the **HandPhysicsService** on Layer 2 where the **Ignore Raycast** layer is.

![image](https://user-images.githubusercontent.com/118937840/203646939-ac827c78-8b3c-4b26-aa63-906a61f9b639.png)

I'm not sure if this is the best way to fix this problem, but it works for now.

**Comment by luffy-yu** [comment]
Same problem. @JosiSiah 's solution works!

**Comment by IssueSyncBot** [comment]
We appreciate your feedback and thank you for reporting this issue.

Microsoft Mixed Reality Toolkit version 2 (MRTK2) is currently in limited support. This means that Microsoft is only fixing high priority security issues. Unfortunately, this issue does not meet the necessary priority and will be closed. If you strongly feel that this issue deserves more attention, please open a new issue and explain why it is important.

Microsoft recommends that all new HoloLens 2 Unity applications use MRTK3 instead of MRTK2.

Please note that MRTK3 was released in August 2023. It features an all-new architecture for developing rich mixed reality experiences and has a minimum requirement of Unity 2021.3 LTS. For more information about MRTK3, please visit [https://www.mixedrealitytoolkit.org](https://www.mixedrealitytoolkit.org).

Thank you for your continued support of the Mixed Reality Toolkit!
